src/main/java/frc/robot/AutoRotateUtil.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.subsystems.Swerve;
+
+public class AutoRotateUtil {
+
+    private final Swerve s_Swerve;
+    private final PIDController pidController;
+
+    private int m_angle;
+
+    public AutoRotateUtil(Swerve s_Swerve, int angle) {
```

Comment:
Can remove Swerve s_Swerve and on line 15 instead use  : this.s_Swerve = Swerve.getInstance;

Reply:
did that here:
https://github.com/CSE-FortOsageCTC/5801_Inspire_2024/pull/26/commits/90ce5b23d15331d66116ab3f8639e61aa9511d54